### PR TITLE
[Executor-benchmark] Add support for pushing metrics to victoriametrics

### PR DESCRIPTION
**Pull request by sitalkedia**
### Description

Will be helpful to get insights from executor benchmark running on laptop. 

### Test Plan

Tested by running the benchmark on laptop

```
2023-03-05T00:58:06.877199Z [main] INFO crates/aptos-push-metrics/src/lib.rs:119 Execution dashboard link: https://o11y.aptosdev.com/grafana/d/execution/execution?from=now-5m&to=now&var-Datasource=VictoriaMetrics+Global&var-metrics_source=All&var-chain_name=executor-benchmark&var-cluster=All&var-namespace=1677977886876947000&var-role=All
```

### CODE REVIEW

**Review by grao1991 — APPROVED** [review]

File: crates/aptos-push-metrics/src/lib.rs [hunk 1]
```diff
@@ -86,20 +116,71 @@ impl MetricsPusher {
...
+            .as_millis()
+            .to_string();
+        let start_time = SystemTime::now()
+            .sub(Duration::from_secs(600))
+            .duration_since(UNIX_EPOCH)
+            .unwrap()
+            .as_millis()
+            .to_string();
+        let mut url =
+            Url::parse("https://o11y.aptosdev.com/grafana/d/execution/execution").unwrap();
```
Comment (ibalajiarun) [inline crates/aptos-push-metrics/src/lib.rs:148]: We could get the base URL from env variable as well and use `o11y.aptosdev.com` as default if env isn't available.

**Review by ibalajiarun — APPROVED** [review]
LGTM. Left a comment.

File: crates/aptos-push-metrics/src/lib.rs (hunk 1 above)
Comment (sitalkedia) [reply crates/aptos-push-metrics/src/lib.rs:148]: Addressed this in a separate PR - https://github.com/aptos-labs/aptos-core/pull/6929

File: crates/aptos-push-metrics/src/lib.rs [hunk 2]
```diff
@@ -86,20 +116,71 @@ impl MetricsPusher {
             "Starting push metrics loop. Sending metrics to {} with a frequency of {} seconds",
             push_metrics_endpoint, push_metrics_frequency_secs
         );
+
+        info!(
+            "Execution dashboard link: {} ",
```
Comment (msmouse) [inline crates/aptos-push-metrics/src/lib.rs:121]: The MetricPusher is used by other tooling as well, it's confusing to print this here.  Can you move these out of the pusher?

File: crates/aptos-push-metrics/src/lib.rs [hunk 3]
```diff
@@ -86,20 +116,71 @@ impl MetricsPusher {
...
+
+        url
+    }
+
+    fn push_metrics_extra_labels(chain_name: &str, namespace: &str) -> Vec<String> {
+        vec![
+            format!("chain_name={}", chain_name),
+            "cluster=unknown".into(),
+            "metrics_source=unknown".into(),
+            "kubernetes_pod_name=unknown".into(),
```
Comment (msmouse) [inline crates/aptos-push-metrics/src/lib.rs:167]: will these mess up with existing usage of the pusher? (The db-backup for example, where these are set from outside)

File: storage/backup/backup-cli/src/bin/db-backup-verify.rs [hunk 4]
```diff
@@ -36,7 +36,7 @@ async fn main() -> Result<()> {
 async fn main_impl() -> Result<()> {
     Logger::new().level(Level::Info).init();
 
-    let _mp = MetricsPusher::start();
+    let _mp = MetricsPusher::start("db-backup-verifiy");
```
Comment (msmouse) [inline storage/backup/backup-cli/src/bin/db-backup-verify.rs:39]: It's confusing to set these as the chain_name...

**Pull request merged**